aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -1064,13 +1064,16 @@ AT_ERROR("triangular_solve: MAGMA library not found in "
   auto b_data = b.data<scalar_t>();
   magma_int_t n = magma_int_cast(A.size(-2), "A.size(-2)");
   magma_int_t nrhs = magma_int_cast(b.size(-1), "b.size(-1)");
+  magma_int_t batch_size = magma_int_cast(batchCount(A), "batchCount");
 
-  if (b.dim() == 2) {
+  // batch_size == 1 implies that:
```

Comment:
this comment is a bit misleading because it means **one of** these two cases is true, not both of them.

also, is the same changes needed with SVD and *det methods?

Reply:
I thought that the way I had put forth the cases meant that both of them couldn’t have been true.

SVD loops serially across batches so, this isn’t an issue.

I haven’t checked *det methods (they use batched getrf). I’ll get back to you with those results by night (it’s a bit late back here)

Reply:
No worries! i'm just curious. Thanks!